### ISSUE TITLE
File to import not found or unreadable - LibSass support?

### ISSUE THREAD

**Issue by ronilaukkarinen** [jsahlen/gulp-css-globbing#20]
Hello. I don't seem to get this to work with libsass. I have `var globbing    = require('gulp-css-globbing');` in requires and this is my styles task:

``` javascript
gulp.task('styles', function() {
  gulp.src(sassFile)

  .pipe(globbing({
      extensions: ['.scss']
  }))
  
  .pipe(sass({
    compass: false,
    bundleExec: true,
    sourcemap: false,
    style: 'compressed',
    debugInfo: true,
    lineNumbers: true,
    errLogToConsole: true
  })) 

  .on('error', handleError('styles'))
  .pipe(prefix('last 3 version', 'safari 5', 'ie 8', 'ie 9', 'opera 12.1', 'ios 6', 'android 4')) //adds browser prefixes (eg. -webkit, -moz, etc.)
  .pipe(minifycss({keepBreaks:false,keepSpecialComments:0,}))
  .pipe(pixrem())
  .pipe(gulp.dest(themeDir + '/css'))
  .pipe(reload({stream:true}));
  });
``` 

Getting:

	[gulp-sass] file to import not found or unreadable: modules/*.*

**Comment by jsahlen** [comment]
What happens if you use `gulp-debug` (with the verbose option) between the globbing and sass steps?

**Comment by ronilaukkarinen** [comment]
It's been a while and now after holidays I noticed it's working at the moment, with or without gulp-scss-globbing, perhaps gulp implemented it or something. As far as I'm conserned, this can be closed if no-one else encounters this issue.

**Comment by ronilaukkarinen** [comment]
False alarm, I had wrong import, my bad. The behaviour is still the same. Does gulp-css-globbing support libsass?

**Comment by jsahlen** [comment]
It doesn't really support anything explicitly – it simply replaces occurences of `@import` in the contents of the stream/buffer, and then Gulp is free to pipe that content further down the chain.

**Comment by mdfragala** [comment]
I'm having the same issue using `gulp-compass`. It's really holding me back; I'm trying to modularize my SASS, but manually typing every `@import` is kind of a hassle. Running the task always yields this error message:
````
error src/assets/sass/main.scss (Line 2: File to import not found or unreadable: layout/*.
Load paths:
  /Users/Mitchell/Documents/Repository/project/src/assets/sass
  /Library/Ruby/Gems/2.0.0/gems/compass-core-1.0.3/stylesheets
  Compass::SpriteImporter)

Compilation failed in 1 files.
````
For some reason, `gulp-css-globbing` doesn't seem to be placing in the files matched by the glob; the globs just stay as they are and get passed on to Compass. Here's the task setup I'm working with (compass is configured in a lazypipe elsewhere).
````
gulp.task('style', function(){
  return gulp.src('./src/assets/sass/**/*.scss')
    .pipe(newer('./app/assets/css/'))
    .pipe(cssGlobbing({
      extensions: ['.scss']
    }))
    .pipe(compass())
    .pipe(autoprefixer())
    .pipe(gulp.dest('./app/assets/css/'))
    .pipe(browser.stream())
});
````

**Comment by jsahlen** [comment]
The plugin doesn't replace in every file — it'll only for the "entry point" file, due to how files and streams relate to each other in Gulp.

**Comment by mdfragala** [comment]
The entry point for what, `gulp` or the plugin? If for `gulp`, the `sass` directory only has a single file `main.scss` to pass through the task. Everything else is partials in subdirectories.

This is the globbing I have in `main.scss':

```
@import 'globals/*';
@import 'layout/*';
@import 'modules/*';
@import 'components/*';
```

**Comment by jsahlen** [comment]
Ah, I see. In that case, your `gulp.src()` function is incorrect – the glob pattern `./src/assets/sass/**/*.scss` matches *all* the files, even in subdirectories, which means `gulp-css-globbing` and `compass` will process all of those files. You probably want `./src/assets/sass/*.scss`, or to be explicit it might be better to just to `./src/assets/sass/main.scss`.

**Comment by mdfragala** [comment]
No, this is returns the same error. Although when I took `gulp-compass` out of the pipe chain, the resulting `main.css` has all the fully globbed imports.
